docs/src/index.md

```diff
@@ -58,90 +63,109 @@ in three different flavors:
 
 Red Hat UBI images are primarily intended for OLM consumption.
 
+All container images are signed and come with SBOM and provenance attestations.
```

Comment:
I think it's worth to add that it's per architecture, since in some of them there could be a slightly different SBOM